components/user_settings/modal/user_settings_modal.tsx

```diff
@@ -223,7 +241,7 @@ class UserSettingsModal extends React.Component {
         }
     }
 
-    updateSection = (section, skipConfirm) => {
+    updateSection = (section: any, skipConfirm: any) => {
```

Comment:
`skipConfirm` here can be a `boolean`.
You might be able to type `section` as well but don't worry about it if you can't.